components/server/ee/src/workspace/gitpod-server-impl.ts

```diff
@@ -1065,70 +1067,8 @@ export class GitpodServerEEImpl extends GitpodServerImpl {
                     return;
                 }
 
```

Comment:
everything deleted below is no longer needed because we are now polling for prebuild finish. See above ☝️